### DISCUSSION TITLE
Sum of Squares optimization

### DISCUSSION THREAD

**Discussion by Naeim-Eb** [casadi/casadi#3193] [Q&A] [answered]
Hi,
what is the compatible sum of squares programming (or semidefinite programming) toolbox with Casadi in Python?
Actually, I could not find any documentation or example of solving semidefinite programming with Casadi. It would be great if anyone can provide me with a simple example.

Best Regards,
Naeim

**Comment by jgillis** [comment, marked as answer]
Hi Naeim, semidefinite programming is currently not supported.
We may re-add it in the future. Closest thing we have at the moment is second order cone
